### PR TITLE
Added MinimumNArgs to required commands in channels.go and team.go

### PR DESCRIPTION
**Pull request by waseem18**
#### Summary
1. Added `MinimumNArgs` to required commands in channels.go and team.go
2. Removed arguments length comparison as `MinimumNArgs` would handle it.

#### Ticket Link
No existing ticket. Relates to code refactoring.

### CODE REVIEW

File: cmd/mattermost/commands/channel.go [hunk 1]
```diff
@@ -352,10 +347,6 @@ func moveChannelsCmdF(command *cobra.Command, args []string) error {
 	}
 	defer a.Shutdown()
 
-	if len(args) < 2 {
-		return errors.New("Enter the destination team and at least one channel to move.")
-	}
```
Comment (saturninoabril) [inline cmd/mattermost/commands/channel.go]: Should add `Args` to `MoveChannelsCmd`?

Comment (waseem18) [reply cmd/mattermost/commands/channel.go]: Looks like I missed it. Will add for this too.

Comment (waseem18) [reply cmd/mattermost/commands/channel.go]: Resolved and squashed the commit 👍

**Comment by saturninoabril** [comment]
Change reviewer to George since Jesse might not be available.

**Pull request merged**